src/effects.js

```diff
@@ -78,7 +78,6 @@ function createTween( value, prop, animation ) {
 }
 
 function defaultPrefilter( elem, props, opts ) {
-	/* jshint validthis: true */
```

Comment:
Analogue rule - http://eslint.org/docs/rules/no-invalid-this yields to many errors, not sure what to do here